### PR TITLE
Remove dissect tokenizing from Traefik Filebeat Access Fileset

### PR DESCRIPTION
**Pull request by sayden**
Solves: https://github.com/elastic/beats/issues/9434

I had a problem as I couldn't make it work with ES7 and I struggling to make it work in ES6 because in ES6 we use `int` as type for many Grok patterns but in ES7 we use `long`. It seems that `long` is detected as a `string` in ES6 and it was giving many troubles.

The problem trying to test this in ES7 is that it just gets frozen and, reviewing logs the only clue is a message saying: `Malformed [mappings] section for type [date_detection], should include an inner object describing the mapping`

### CODE REVIEW

**Comment by sayden** [comment]
Pushing to see if it's an issue with my local. I have tested the pipeline successfully in ES6 but I couldn't try with ES7

**Comment by sayden** [comment]
jenkins, test this please

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -215,6 +215,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add convert_timezone to nginx module. {issue}9839[9839] {pull}10148[10148]
 - Add support for Percona in the `slowlog` fileset of `mysql` module. {issue}6665[6665] {pull}10227[10227]
 - Added support for ingesting structured Elasticsearch audit logs {pull}10352[10352]
+- Remove dissect tokenizing from Filebeat Traefik 'access' Fileset. {pull}10442[10442]
```
Comment (ruflin) [inline CHANGELOG.next.asciidoc]: This should go under breaking changes.
```suggestion
- Move dissect pattern for traefik.access fileset from Filbeat to Elasticsearch. {pull}10442[10442]
```

File: filebeat/module/traefik/access/ingest/pipeline.json [hunk 2]
```diff
@@ -3,9 +3,9 @@
     "processors": [
         {
             "grok": {
-                "field": "traefik.access.message",
+                "field": "message",
                 "patterns": [
-                    "(?:%{NUMBER:http.response.body.bytes:long}|-)( (?:\"%{DATA:http.request.referrer}\"|-)?( (?:\"%{DATA:traefik.access.agent}\"|-)?)?( (?:%{NUMBER:traefik.access.request_count:long}|-)?)?( (?:\"%{DATA:traefik.access.frontend_name}\"|-)?)?( \"%{DATA:traefik.access.backend_url}\")?( %{NUMBER:temp.duration:long}ms)?)?"
+                    "%{IPORHOST:source.address} %{GREEDYDATA:traefik.access.user_identifier} %{GREEDYDATA:user.name} \\[%{HTTPDATE:traefik.access.time}\\] \"%{WORD:http.request.method} %{DATA:url.original} HTTP/%{NUMBER:http.version}\" %{NUMBER:http.response.status_code:long} (?:%{NUMBER:http.response.body.bytes:long}|-)?( (?:\"%{DATA:http.request.referrer}\"|-)? (?:\"%{DATA:traefik.access.agent}\"|-)? (?:%{NUMBER:traefik.access.request_count:long}|-)? (?:\"%{DATA:traefik.access.frontend_name}\"|-)? (?:\"%{DATA:traefik.access.backend_url}\"|-)?( %{NUMBER:temp.duration:long}ms|-)?)?"
```
Comment (ruflin) [inline filebeat/module/traefik/access/ingest/pipeline.json]: We should use the dissect processor here as we did in Filebeat for the first part: https://www.elastic.co/guide/en/elasticsearch/reference/master/dissect-processor.html It's more efficient then grok.

File: CHANGELOG.next.asciidoc (hunk 1 above)
Comment (sayden) [reply CHANGELOG.next.asciidoc]: Moved with the suggested message :)

File: filebeat/module/traefik/access/ingest/pipeline.json (hunk 2 above)
Comment (sayden) [reply filebeat/module/traefik/access/ingest/pipeline.json]: Ok, fixed adding a dissect processor at the top of the pipeline

**Comment by sayden** [comment]
jenkins, test this please

**Pull request merged**